sdk/core/Azure.Core/api/Azure.Core.net5.0.cs

```diff
@@ -398,6 +400,16 @@ public abstract partial class ClientOptions
         public static bool operator !=(Azure.Core.ContentType left, Azure.Core.ContentType right) { throw null; }
         public override string ToString() { throw null; }
     }
+    public abstract partial class DelayStrategy
+    {
+        protected DelayStrategy(Azure.Core.RetryMode? mode = default(Azure.Core.RetryMode?), System.TimeSpan? initialDelay = default(System.TimeSpan?), System.TimeSpan? maxDelay = default(System.TimeSpan?)) { }
+        public static Azure.Core.DelayStrategy CreateExponentialDelayStrategy(System.TimeSpan? initialDelay = default(System.TimeSpan?), System.TimeSpan? maxDelay = default(System.TimeSpan?)) { throw null; }
+        public static Azure.Core.DelayStrategy CreateFixedDelayStrategy(System.TimeSpan? delay = default(System.TimeSpan?)) { throw null; }
+        public static Azure.Core.DelayStrategy CreateSequentialDelayStrategy(System.Collections.Generic.IEnumerable<System.TimeSpan>? sequence = null) { throw null; }
+        public System.TimeSpan? GetClientDelayHint(Azure.Response? response, int attempt) { throw null; }
```

Comment:
Should we add a GetServerDelayHint for symmetry? Right now this is being made public with ResponseHeaders.RetryAfter.